apps/client/src/components/qna/QnaPostCard/index.tsx

```diff
@@ -11,18 +11,16 @@ import type { Qna } from '@/types/qna';
 type QnaPostingCardProps = {
```

Comment:
```suggestion
type QnaCardProps = {
```

QnaPostingCard말고 QnaCard는 어떨까요? Posting은 굳이 쓰지 않아도되는 변수명 같다고 생각이 되어서요